_tutorials/bundle.md

```diff
@@ -38,7 +38,7 @@ var resourceImagePng = &fyne.StaticResource{
 	}}
 ```
 
-As you see the default naming is "resource\<Name\>.\<Ext\>". The name and package used in this file can be customised in command parameters. We can then use this name to,
```

Comment:
Don’t you think it’s clearer with the “.” Separating name and extension?

Reply:
Is this intentional? The variable name `resourceImagePng` did not include a period, so I just thought that it is mistake.

Reply:
Oh goodness you are correct, my apologies